README.rst

```diff
@@ -5,8 +5,8 @@ mitmproxy
 
 This repository contains the **mitmproxy** and **pathod** projects.
 
-``mitmproxy`` is an interactive, SSL-capable intercepting proxy with a console
-interface.
+``mitmproxy`` is an interactive, TLS/SSL-capable intercepting proxy with a console
+interface for HTTP/1, HTTP/2, and WebSockets.
```

Comment:
-1 to the "for HTTP/1, HTTP/2, and WebSockets." addition, IMHO the value/additional characters ratio isn't high enough.
Also I think we should either do "TLS" or "SSL/TLS".